src/Hl7.Fhir.Core.Tests/Rest/FhirClientTests.cs

```diff
@@ -1092,16 +1092,42 @@ public void TestOperationEverything()
             loc = client.TypeOperation<Patient>("everything", null, false);
             Assert.IsNotNull(loc);
 
-            // GET operation $everything with 1 parameter
-            // This doesn't work yet. When an operation is used with primitive types then those parameters must be appended to the url as query parameters.
-            // loc = client.TypeOperation<Patient>("everything", new Parameters().Add("start", new Date(2017, 10)), true);
-            // Assert.IsNotNull(loc);
+            
+
+            // GET operation $everything with 1 primitive parameter
+             loc = client.TypeOperation<Patient>("everything", new Parameters().Add("start", new Date(2017, 11)), true);
+            Assert.IsNotNull(loc);
+
+            // GET operation $everything with 1 primitive2token parameter
+            loc = client.TypeOperation<Patient>("everything", new Parameters().Add("start", new Identifier("", "example")), true);
```

Comment:
Klein stylistisch puntje:  we proberen binnen de API parameters (vaak primitieve waarden) waarvan het niet snel duidelijk is wat ze doen (vaak optionele booleans) door de naam vooraf te laten gaan:

client.TypeOperation<Patient>(........., useGet: true);

Reply:
Ah dat wist ik niet, maar mooie oplossing